src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,35 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
Please ensure that you don't start the class implementation with an empty line. Remove the unnecessary empty lines before the class declaration.